staging/src/k8s.io/apimachinery/pkg/labels/selector_test.go

```diff
@@ -156,10 +156,9 @@ func TestSetMatches(t *testing.T) {
 	expectMatchDirect(t, Set{"baz": "blah"}, labelset)
 	expectMatchDirect(t, Set{"foo": "bar", "baz": "blah"}, labelset)
 
-	//TODO: bad values not handled for the moment in SelectorFromSet
-	//expectNoMatchDirect(t, Set{"foo": "=blah"}, labelset)
-	//expectNoMatchDirect(t, Set{"baz": "=bar"}, labelset)
-	//expectNoMatchDirect(t, Set{"foo": "=bar", "foobar": "bar", "baz": "blah"}, labelset)
+	expectNoMatchDirect(t, Set{"foo": "lah"}, labelset)
```

Comment:
This change doesn't fix the TODO, and shouldn't remove it or the commented out test cases.

Reply:
When the value or key of Set is invalid, the func SelectorFromSet will return internalSelector{},bad values not handled for the moment in SelectorFromSet. Should we add an err in func SelectorFromSet return?